media/css/base-cloud9.css

```diff
@@ -169,7 +169,7 @@ body.vscode-dark .settings-panel {
 body.vscode-light .settings-panel {
     background: rgba(var(--shade), 0.02) !important;
 }
-#invoke-button-container {
+.button-container {
     border: none !important;
     background: none !important;
```

Comment:
Is this still needed?

Reply:
No, I made the stylesheet simpler in https://github.com/aws/aws-toolkit-vscode/pull/2594/files#diff-8d6618221c6141aac8234414339d512b381ad6d97a34ed15d1788b4a67ac82c2